include/swift/AST/Attr.def

```diff
@@ -311,12 +323,13 @@ CONTEXTUAL_DECL_ATTR(weak, ReferenceOwnership,
   OnVar |
   DeclModifier |
   NotSerialized |
+  // Reference ownership stability checked separately
```

Comment:
I guess it's only ABI for fixed layout structs. Do we check that?

Reply:
Hm, not sure. That can be handled elsewhere though.

[This is `weak`]